packages/terra-data-grid/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.1.0 - (November 9, 2023)
+
 * Fixed
   * Fixed the column header background color for the Orion Fusion theme.
   * Fixed pinned column divider issue during keyboard resize.
```

Comment:
Should these packages with fixes be a patch release instead of a minor release?

Reply:
There is some stuff added as well like the ability to select a range of cells to FlowsheetDataGrid, so I will keep this a minor version